### ISSUE TITLE
Extending the guide for setting up the same using bottles

### ISSUE THREAD

**Issue by joeljosephreji** [nxjosephofficial/primevideo-linux#13]
It would be beneficial if the guide could be extended for running Google Chrome via bottles for prime video.

**Comment by nxjosephofficial** [comment]
Good idea but I couldn't make it run. Bottles does not have pure `wine-staging` and I tried with `Soda 7.0.9` and `Vaniglia-7.18` which includes staging patches but they did not work. You can install `wine-staging` with your package manager and select it (it may seem as sys-wine-staging or sys-wine) in Bottles.

**Comment by joeljosephreji** [comment]
Ok, let me try from my side. Thanks for the inputs!

**Comment by Mastermindaxe** [comment]
@joeljosephreji Any luck yet? I'd be interested in running this in bottles
